charter.md

```diff
@@ -0,0 +1,336 @@
+```
+
+    (source: https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#responsibilities--empowerment-of-tags)
+
+- Charter / Description of WG (_This document once it's merged_)
+- Time Box:
+  - 2 quarters (Q1-Q2 2022)
+- Deliverables:
+  - github.com/observe-k8s: An umbrella organization for repos that realize the goals of this org
+  - Repositories containing demo code and examples on how to run and utilize observability tools
+  - Examples of use cases we mentioned from whitepaper
+  - Catalog of resources for learning about observability
+
+### Interactive Demo environment & playground
+
+- Rather than a scripted tutorial or guide, allow free experimentation.
+- Demonstrate _value_ of various solutions by (clonable, reproducible) examples. In other words “see it first, then easily replicate in your own env”
+- Enable multiple views of the same underlying infra/workloads. As the TAG isn’t meant to be a “kingmaker” -
+- a site leveraging Angular (as a framework in particular) might make this somewhat simpler to achieve. Caveat: I’m not a front end dev :)
+- Prometheus provides a demo service [here](https://prometheus.demo.do.prometheus.io/) - anecdotally it is really helpful in explaining key concepts quickly in slack / other support requests.
+  - Thanos is discussing adding one too [here](https://github.com/thanos-io/thanos/issues/4606).
+
+### Design: observe.io
+
+- visualization / details showing the actual request, and the infra that's generating the response. Useful as a doorway to engage with persons that find their way to one of the 2 main front doors (observe-k8s.dev, observe-k8s.io).
+
+          "Welcome! This is you and here is the web server that rendered the site you're currently observing :)  \
+
+  \
+  Enter! See what else is running!" \
+
+- For 2d (flat browser) and 3d (AR, MR, VR, 3d rendering in-browser) the site would show other CNCF Projects running, hosted by the CNCF Lab ([https://github.com/cncf/cluster](https://github.com/cncf/cluster))
+- Suggest [https://angular.io](angular.io)
+  - extensibility via drop-in Angular Controls
+  - modelling connections to back end API’s ([Services](https://angular.io/guide/architecture-services))
+  - shared data model enabling n views of same underlying data simply.
+
+### Design: observe.dev
+
+- Targeting developer persona
+
+###
+
+### CNCF interested parties and collaboration opportunities:
+
+#### SIG Instrumentation
+
+- Source of upstream k8s signals
+
+#### CNCF TAG App Deploy: GitOps WG
+
+- Open Standard for data format for reporting
+
+\
+TAG Observability.
+
+- How to visualize execution of Operators, Controllers en masse / aggregate.
+- Engage with SIG Instrumentation re: gaps and opportunities for signals from k8s
+- Collect and Curate templates, recipes, examples, demos highlighting the observation of cloud native systems and workloads (CNCF Projects).
+
+## Personas
```

Comment:
I agree that this section is important but as it stands it's not very useful. Let's either cut it down to "Gibbs is on it" or flesh it out.

Reply:
done